terraform/paas/production.env.tfvars

```diff
@@ -0,0 +1,26 @@
+paas_space                = "get-into-teaching-production"
+paas_database_common_name = "school-experience-prod-pg-common-svc"
+paas_redis_1_name         = "school-experience-prod-redis-svc"
+paas_application_name     = "school-experience-app-production"
+paas_internet_hostnames   = []
+application_instances     = 1
+delayed_jobs              = 1
+environment               = "production"
+application_environment   = "dfe-school-experience-production"
+azure_key_vault           = "s105t01-kv"
+azure_resource_group      = "s105t01-staging-vault-resource-group"
+database_plan             = "small-ha-11"
+redis_1_plan              = "small-ha-5_x"
```

Comment:
Have you looked at micro or tiny plans? They are cheaper and may be enough

Reply:
See: https://admin.london.cloud.service.gov.uk/marketplace/754c7e96-bc62-45ca-8fe7-a5064d922588

Reply:
Looks good will try